documentation/api/io.strimzi.api.kafka.model.authentication.KafkaClientAuthenticationCustom.adoc

```diff
@@ -0,0 +1,90 @@
+To configure custom authentication, set the `type` property to `custom`.
```

Comment:
Not sure we need this file. If I understand correctly, this is custom Kafka listener/authentication option. In which case, this be added under the `authentication` heading in _documentation/api/io.strimzi.api.kafka.model.listener.arraylistener.GenericKafkaListener.adoc_

Reply:
Woops, that was an accident on my behalf! I renamed + linked from `documentation/api/io.strimzi.api.kafka.model.listener.arraylistener.GenericKafkaListener.adoc`